examples/basic_mysql_connection/index.js

```diff
@@ -12,12 +12,12 @@ var connection = mysql.createConnection({
 });
 
 connection.connect();
-let query=['select count(*) as items from todos',
-           'select * from todos where done',
-           'select * from todos where !done',
-					 'select * from todos order by due DESC',
-					 'select * from todos where due=(select max(due) from todos)',
-					 'select * from todos where name like "%databases%"'
+let query=['SELECT COUNT(*) as items FROM todos',
+           'SELECT * FROM todos WHERE done',
+           'SELECT * FROM todos WHERE !done',
+					 'SELECT * FROM todos GROUP BY DUE DESC',
```

Comment:
Did you mean to use `ORDER BY`? Also, this will retrieve them ordered by when they're due, but not when they're added like the question asked.